server/utils/startup.ts

```diff
@@ -50,10 +50,11 @@ export async function checkMigrations() {
     return;
   }
 
+  const isProduction = env.ENVIRONMENT === "production";
   const teams = await Team.count();
   const providers = await AuthenticationProvider.count();
 
-  if (teams && !providers) {
+  if (isProduction && teams && !providers) {
```

Comment:
I was seeding the dev environment and that exception came up. @tommoor suggested to add this workaround for now.